_posts/2021-12-14-weaving-the-geospatial-fabric.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Weaving the Fabric - Building Authoritative Geospatial Datasets through Collaboration, Relationships, and Data Standards
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2021-12-14 10:01:46
+categories:
+  - Featured
+tags:
+  - sgid
+  - gisac
+  - collaboration
+  - standards
+---
+
+![Weaving the Fabric]({% link images/weaving_the_fabric.png %})
+{: .flex .flex--center}
+
+### Many Hands Make Light Work
+{: .text-left}
+
+It’s been said that it took about [one-hundred-thousand](https://www.historymuseum.ca/cmc/exhibitions/civil/egypt/egca12e.html#:~:text=The%20Greek%20historian%20Herodotus%20was,creates%20a%20very%20stable%20structure.) humans to build the Pyramids of Giza and a [similar number](https://www.peoplesworld.org/article/today-in-labor-history-panama-canal-built-by-75-000-opens/) to build the Panama Canal. Prior to the internet, large-scale collaboration is generally thought to be limited to that number. Any more, and it became difficult to manage. What made these projects successful is that they were well-organized and that they adhered to a set of standards. Today we are able to collaborate with an infinite number of people, however, organization and standards are still limiting factors.
```

Comment:
```suggestion
It’s been said that it took about [100,000](https://www.historymuseum.ca/cmc/exhibitions/civil/egypt/egca12e.html#:~:text=The%20Greek%20historian%20Herodotus%20was,creates%20a%20very%20stable%20structure.) humans to build the Pyramids of Giza and a [similar number](https://www.peoplesworld.org/article/today-in-labor-history-panama-canal-built-by-75-000-opens/) to build the Panama Canal. Prior to the internet, large-scale collaboration is generally thought to be limited to that number. Any more, and it became difficult to manage. What made these projects successful is that they were well-organized and that they adhered to a set of standards. Today we are able to collaborate with an infinite number of people, however, organization and standards are still limiting factors.
```

This looks really good! Love the use of the logo!

One nitpick, I think the standard convention is that any number under ten is spelled out and any number 10 and over is written using numerals instead of spelled out. But I could be wrong.